README.md

```diff
@@ -9,3 +9,22 @@ See https://github.com/nation3/agreements-app
 ## Documentation
 
 https://github.com/nation3/agreements-app/blob/main/README.md#documentation
+
+## How to deploy/redeploy subgraph
+Update `networks.json` file before deploying subgraph
+
+#### Mainnet
+```
+yarn auth
+yarn codegen
+yarn build-mainnet 
```

Comment:
Is it possible to also add these 3 steps to a GitHub workflow? - https://github.com/nation3/agreements-contracts/tree/main/.github/workflows

Reply:
Yeah, we can, but it's useless since yarn auth needs an access token from Subgraph. I don't think we should publish this token and yarn codegen just creates a schema.ts file.